src/content/docs/en/installation/getting.mdx

```diff
@@ -1,67 +1,49 @@
 ---
 
 import Alert from "../../../../components/Docs/Alert.astro";
 
-Thank you for choosing Ultramarine Linux!
-This guide will have you up and running in no time at all!
-
-## System requirements
-
-- 4GB of RAM (8GB or more recommended)
-- 64-bit processor (x86_64/amd64) or an AArch64 processor (ARM64)
-- 10GB of free disk space (20GB or more recommended)
-- Internet connection (for downloading the ISO image, updates, and additional drivers)
-- A computer that supports booting from a USB drive or DVD (most computers made after 2007 should support this feature)
-- A DVD or USB drive (for creating the bootable media)
-- Basic-to-intermediate computer knowledge
-
-### Additional requirements for specific hardware
-
-- For computers with NVIDIA graphics cards, you are strongly recommended to install additional drivers after installation. This can be done by opening the "Software" app and searching for "NVIDIA". You can also install the drivers by running `sudo dnf install nvidia-driver` in the terminal.
-- For computers with AMD or Intel graphics cards, Ultramarine Linux should support your graphics card out of the box, as part of the Linux kernel.
-- Some computers with specific Broadcom or Realtek chipsets may require additional drivers to be installed. You may need to connect to the internet via Ethernet or USB tethering from your phone to install these drivers.
-- For computers with a TPM (Trusted Platform Module), you can enable it in the BIOS settings. However driver and kernel support for TPM may not work as expected. We recommend you disable Secure boot before installing Ultramarine Linux, then experiment with enabling it after installation.
-
-## Downloading
-
-### Choosing your edition
+## Choosing Your Edition
 
 Ultramarine Linux comes in 4 variants:
 
 - **Flagship Edition**: Our default and most popular variant. Choose Flagship if you'd like a familiar and stylish experience.
+- **KDE Edition**: Our second most popular variant. Choose KDE if you'd like a simply customisable experience.
 - **GNOME Edition**: Elegant and Modern. Choose GNOME if you'd like a simple, unique experience.
-- **Pantheon Edition**: Our second most popular variant. Choose Pantheon if you'd like a thoughtful and capable experience.
-- **KDE Edition**: Our newest variant. Choose KDE if you'd like a simply customisable experience.
+- **Pantheon Edition**: Thoughtful and Graceful. Choose Pantheon if you'd like a refined experience.
+
+Check the [hardware page](hardware) to make sure your hardware is supported.
 
-Ultramarine Linux supports 64bit Intel, AMD, and ARM computers.
+Ultramarine Linux is distributed with ISO files available for download from [our website](https://ultramarine-linux.org).
 
 <Alert type="warning">
-  Despite supporting ARM computers, Ultramarine Linux **_DOES NOT_** support M1
-  and M2 Macs.
+  Fyra Labs cannot guarantee the safety or quality of images downloaded from
+  sites other than our website or FOSSTorrents
 </Alert>
 

```

Comment:
```suggestion
  Fyra Labs cannot guarantee the safety or quality of images downloaded from
  sites other than our website or FOSSTorrents.
  It is also recommended to verify the checksums of the downloaded ISOs. See the section below.
</Alert>


```

Reply:
I don't think we need to add that the section below already suggests it

Reply:
owk